### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by yinlinchen**
**JIRA Ticket**: ([link](https://webapps.es.vt.edu/jira/browse/LIBTD-2591)) (:star:)

Update readme to the latest version.

### CODE REVIEW

**Review by whunter — APPROVED** [review]
This looks great to me. My only question would be whether we should use the "regular" JSON instead of the the Dynamo JSON for the examples/site.json file. The Dynamo version is kind of complicated and hard to follow for me.

I still approve the PR though, it's just a thought.

**Comment by yinlinchen** [comment]
> This looks great to me. My only question would be whether we should use the "regular" JSON instead of the the Dynamo JSON for the examples/site.json file. The Dynamo version is kind of complicated and hard to follow for me.
> 
> I still approve the PR though, it's just a thought.

@whunter Good suggestion. I added both format.
